rotkehlchen/db/eth2.py

```diff
@@ -563,6 +562,11 @@ def process_accumulating_validators_balances_and_pnl(
 
         from_ts_ms, to_ts_ms = ts_sec_to_ms(from_ts), ts_sec_to_ms(to_ts)
         validator_balances: dict[int, FVal] = defaultdict(lambda: ZERO)
+        for v_index in validator_indices:  # Get the latest balance from cache
+            if cached_balances_over_time.get(v_index) is not None:
+                latest_cached_ts = max(cached_balances_over_time[v_index].keys())
```

Comment:
Would it be possible to not have to get all keys and just get last one? Only if `cached_balances_over_time[v_index]` is always worted at this point. Is it a sorted dict or nah?